test/image/compare_pixels_test.js

```diff
@@ -35,15 +40,34 @@ function runAll () {
          * More info:
          * https://github.com/plotly/plotly.js/issues/62
          *
-         * 40 test cases are removed:
+         * 41 test cases are removed:
          * - font-wishlist (1 test case)
          * - all gl2d (38)
-         * - gl2d_bunny-hull (1)
+         * - gl3d_bunny-hull (1)
+         * - polar_scatter (1)
          */
-        t.plan(files.length - 40);
+        var mocks = allMocks.filter(function(mock) {
+            return !(
+                mock === 'font-wishlist.json' ||
+                mock.indexOf('gl2d') !== -1 ||
+                mock === 'gl3d_bunny-hull.json' ||
+                mock === 'polar_scatter.json'
+            );
+        });
+
+        var BASE_TIMEOUT = 500,  // base timeout time
+            BATCH_SIZE = 5,      // size of each test 'batch'
+            cnt = 0;
+
+        function testFunction() {
+            testMock(mocks[cnt++], t);
+        }
+
+        t.plan(mocks.length);
 
-        for (var i = 0; i < files.length; i ++) {
-            testMock(files[i], t);
+        for(var i = 0; i < mocks.length; i++) {
+            setTimeout(testFunction,
+                BASE_TIMEOUT * Math.floor(i / BATCH_SIZE) * BATCH_SIZE);
```

Comment:
Man - I really assumed we were already batching them. Can't believe we went this long without failures! Good to know the image_server can work with such a huge onslaught :)